internal/portforward/portforward.go

```diff
@@ -84,21 +86,27 @@ func localPortsHandler(fw *portforward.PortForwarder, opts Options) {
 		go func() {
 			select {
 			case <-opts.ReadyChannel:
-				ports, err := fw.GetPorts()
+				forwardedPorts, err := fw.GetPorts()
 				if err != nil {
-					// TODO: alert user (GH#497)
+					(*alerter).SendAlert(action.CreateAlert(action.AlertTypeError, "Error resolving local port: "+err.Error(), action.DefaultAlertExpiration))
```

Comment:
No need for this once you convert the argument to just taking the interface not a pointer to the interface.
You probably want to check if `alerter` is nil to avoid a panic output that could happen if this go routine was finishing up when Octant was interrupted / killed.